graphql_compiler/tests/test_helpers.py

```diff
@@ -278,7 +278,6 @@ def generate_schema(graph_client):
     schema_data = [x.oRecordData for x in schema_records]
     type_overrides = {
         "UniquelyIdentifiable": {"uuid": GraphQLID},
-        "Animal": {"birthday": GraphQLDate},
         "Event": {"event_date": GraphQLDateTime}
```

Comment:
How about the `DateTime` one below? Is that one not inferred automatically?

Reply:
It is inferred as Date.

Reply:
We need it to be inferred as Datetime for other tests.

Reply:
Let's change its definition in the schema from `CREATE PROPERTY Event.event_date Date` to `CREATE PROPERTY Event.event_date DateTime` then, otherwise our database and GraphQL schemas will disagree and we might get invalid compiled output.